#! /bin/sh /usr/share/dpatch/dpatch-run
## 50_relative_paths_in_htmldoc.dpatch by Henning Glawe <glaweh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use htmldir instead of htmlroot to generate relative paths

@DPATCH@
diff -urNad pdl-2.4.2/Doc/mkhtmldoc.pl /tmp/dpep.CQdnZX/pdl-2.4.2/Doc/mkhtmldoc.pl
--- pdl-2.4.2/Doc/mkhtmldoc.pl	2001-08-06 11:36:07.000000000 +0200
+++ /tmp/dpep.CQdnZX/pdl-2.4.2/Doc/mkhtmldoc.pl	2005-01-15 17:05:02.000000000 +0100
@@ -193,7 +193,7 @@
     my $verbopts = $verbose ? "--verbose" : "--quiet";
     pod2html("--podpath=PDL:.",
 	     "--podroot=$topPerlDir",
-	     "--htmlroot=$htmlrootdir",
+	     "--htmldir=$htmlrootdir",
 	     "--libpods=perlfaq",
 	     "--recurse",
 	     "--infile=$file",
